### PR TITLE
AT_030.002 | Footer > After clicking on the `Website terms and conditions` user navigated to a new pdf file page

### PR DESCRIPTION
**Pull request by kgoldpasv**
AT_030.002 | Footer > After clicking on the Website terms and conditions user navigated to a new pdf file page

### CODE REVIEW

**Review by Penshin — DISMISSED** [review]
All test cases should be independent. In this case, the second test case depends on the first one.

**Review by MarinaMone — DISMISSED** [review]
Looking good

**Comment by kgoldpasv** [comment]
@Penshin   Stan, please approve again. I changed my code according to your suggestion. Do I need refactoring, or just make a pull request without any changes?

**Comment by Penshin** [comment]
You need to commit the changes and push them to GitHub. The changes will automatically connect to this PR and your file will be updated

File: cypress/e2e/arkGroup.cy.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/// <reference types = "cypress" />
+
+describe('all ARK team tests', () => {
+          
+      
+    it('AT_030.001|Footer>Verify redirection to terms and conditions', function() {
+        cy.visit('https://openweathermap.org/')
+    })
```
Comment (MariaMoroz) [inline cypress/e2e/arkGroup.cy.js:8]: объедините пожалуйста два it в один. У нас на проекте пока так: один тест кейс = один it

**Review by Penshin — DISMISSED** [review]

**Review by Penshin — APPROVED** [review]

**Review by StanislavBliss — APPROVED** [review]

**Pull request merged**